content/doc/book/managing/system-properties.adoc

```diff
@@ -55,407 +55,488 @@ dd {
 </style>
 ++++
 
-`hudson.ClassicPluginStrategy.useAntClassLoader`::
+[[hudson_ClassicPluginStrategy_useAntClassLoader]]
+<<hudson_ClassicPluginStrategy_useAntClassLoader,`hudson.ClassicPluginStrategy.useAntClassLoader`>>::
```

Comment:
I don't think this has the result that you want.  It makes the text a hyperlink to an ID that is immediately before the text in the file.  Thus, it is a hyperlink to itself.  Is that intentional so that someone that wants to link to it can click it to get the link?
```suggestion
<<hudson_ClassicPluginStrategy_useAntClassLoader,`hudson.ClassicPluginStrategy.useAntClassLoader`>>::
```

The navigation to these labels with Chrome on my Windows computer causes the clicked link to "hide itself" under the top black bar.  Same problem exists in the glossary.  I would love to have a fix for it, but don't know how to fix it.  I assume it is some interaction between the web browser and the top bar.

### Before clicking the "hudson.lifecycle" link

![Annotation 2020-05-28 064653](https://user-images.githubusercontent.com/156685/83148013-a7070000-a0b5-11ea-953c-85e88366a5d7.png)

### After clicking the "hudson.lifecycle" link

![Annotation 2020-05-28 073636](https://user-images.githubusercontent.com/156685/83148383-209eee00-a0b6-11ea-9af1-454583cb97a5.png)

Unfortunately, I truly don't know how to fix that.  Chrome is probably scrolling the page so that the link is at the top of the page, but that hides it under the top bar.

Reply:
@MarkEWaite, the self-link was indeed added as per @timja's review comment and our discussion above. I know it's a bit odd to have such a link but that's currently the only way for the user to find out the hyperlink and bookmark it. Otherwise they have to know or read the .adoc file to discover it.

I do like how a link symbol appear on hover for the section titles when `:sectlinks:` is set. But I'm not sure how that can be done easily. And even with `:sectlinks:` the section title is an actual hyperlink to itself. The default Asciidoctor stylesheet then displays linked section titles with the same color and styles as unlinked section titles, so that it's not obvious.

I have also seen the issue when you click on a link that the first part is hidden below the top bar. That is a general issue for the whole site. I can create a separate issue to investigate that.

Reply:
It seems that a similar issue was resolved for cross page anchors (https://github.com/jenkins-infra/jenkins.io/pull/679) and that there are known solutions to the in-page anchors overlapping with the fixed top bar. I'll have a look.

Reply:
Thanks for the clarification.  I like the convenience of the self-link.  

It might help readers like me if there were an explanatory comment at the start of the document that describes the technique and how it helps.  That's entirely optional, since the pattern is quite clear when viewing the text of the file.